docs/hadoop.md

```diff
@@ -0,0 +1,48 @@
+# `hadoop` Metrics Receiver
+
+The `hadoop` metrics receiver can fetch stats from a Hadoop server's Java Virtual Machine (JVM) via [JMX](https://www.oracle.com/java/technologies/javase/javamanagement.html).
+
```

Comment:
Just realized that Hadoop has some pending discussions going on in the `Category 3 - Regional metrics and master metrics` section of the `Duplicate / Overlapping Metrics Issue from Clustered Applications` doc.

We'll need to resolve that conversion before releasing this.

Reply:
Per the recent decision in the Duplicate / Overlapping Metrics Issue from Clustered Applications, let's add some context here:

> It collects metrics specific to the local region server, as well as metrics presented by the Master node if the node being monitored is indeed the Master. For High Availability configurations, it is recommended for every master node to report cluster metrics, even though they are identical, to avoid single point of failures when one master goes down.

Reply:
Added